### DISCUSSION TITLE
The result "" of evaluating the value "$(AvaloniaBuildServicesTasksLocation)" of the "AssemblyFile" attribute in element <UsingTask> is not valid.  D:\Programming\VSNet\###\packages\Avalonia.BuildServices.0.0.16\build\Avalonia.BuildServices.targets

### DISCUSSION THREAD

**Discussion by GronbergInfo** [AvaloniaUI/Avalonia#11964] [Q&A] [answered]
Just installed Avalonia + Avalonia.Desktop thru nuget manager.
Then i get an update-counter for Avalonia.Angle.Windows.Native and Avalonia.BuildServices.

When i try to install/update these to i get the title error and nuget fails installing/updating.

Just installed The Avalonia.Angle.. alone, it worked, but Avalonia.BuildServices still fails.

**Comment by timunie** [comment]
better to start with a new project using the templates or template studio. Normally you don't need to install / update depended packaged on your own. 

Hint: if you are just at the beginning, consider to directly use 11.0 RC versions: https://docs.avaloniaui.net/docs/next/welcome 

Install templates: https://docs.avaloniaui.net/docs/next/get-started/install

**Comment by GronbergInfo** [comment]
> better to start with a new project using the templates or template studio. Normally you don't need to install / update depended packaged on your own.
> 
> Hint: if you are just at the beginning, consider to directly use 11.0 RC versions: https://docs.avaloniaui.net/docs/next/welcome
> 
> Install templates: https://docs.avaloniaui.net/docs/next/get-started/install

I just installed in a new WPF project, that i have done absolutely nothing in (been concentrating on the windows service part of the solution).
Had to revert changes via github to remove all dependencies.

**Comment by timunie** [comment, marked as answer]
well Avalonia is not WPF. It's similar, but not a lib to extend WPF: So either do an Avalonia App or a WPF App. You can host WPF controls in Avalonia and the other way around, but this is next level ^^

**Comment by GronbergInfo** [comment]
> well Avalonia is not WPF. It's similar, but not a lib to extend WPF: So either do an Avalonia App or a WPF App. You can host WPF controls in Avalonia and the other way around, but this is next level ^^

Ah, oki, that probably explains my problem, i was looking for wpf ui's, but if the avalonia app is better suited, and similar in terms of programming and such, i could just use the A-App since i havent started on mu GUI part of solution


EDIT: Ah, i see the extension only supports VS up to 2019 and not the community 2022

**Reply by timunie** [reply]
there is an extension for VS2022. But it has another name. https://marketplace.visualstudio.com/items?itemName=AvaloniaTeam.AvaloniaVS

**Reply by GronbergInfo** [reply]
Thanks you, again. Now i am probably set to start the project.

**Comment by timunie** [comment]
Again, please read the docs I've linked. In Addition, here are some samples to get started. https://github.com/AvaloniaUI/Avalonia.Samples 

I'll move this issue into a Q&A now as I think it's not a bug. 

Last but not least, if you want to go with Avalonia you are welcome to [join us on Telegram](https://t.me/Avalonia)
